app/services/live_stream_updater.rb

```diff
@@ -67,15 +67,25 @@ def presenter
     CoronavirusPagePresenter.new(live_content_item["details"], "/coronavirus")
   end
 
-  def live_stream_payload
+  def payload
     presenter.payload.merge(
       {
         "title" => "Coronavirus (COVID-19): what you need to do",
         "description" => live_content_item["description"],
+        "details" => live_content_item["details"].deep_merge(live_stream_payload),
```

Comment:
Previously `presenter.payload` on line 71 contained the livestream content. Now it's removed from the presenter it needs to be added in here. Added benefit is it will be easier to remove in the future because all the livestream updater stuff is in one class.